### ISSUE TITLE
Gibberish audio on preliminary training

### ISSUE THREAD

**Issue by Pranjalya** [sh-lee-prml/HierSpeechpp#30]
Thanks for the repo. I had around 30 hours of custom Hindi data which I wanted to train and test the model on. Training only the TTV part on 4x A6000 GPUs with 64 batch size, I tried inferencing with the provided VC checkpoint, but I was getting unintelligible results. There was no distortion audio, but just unintelligible speech.
What do you reckon could be the cause of it? Was it because I will need a VC model for Hindi as well, or my training steps were low for getting the results? And after how many steps, do you think we can expect a checkpoint on which we can get a decent preliminary result?
Thanks again!

**Comment by hayeong0** [comment]
What does 'unintelligible speech' mean? Can I see your training logs or Tensorboard? 

We have used about 60 hours of Hindi data from LIMMITS and have experience using phonemizer. Have you checked if tokens were properly extracted during the training and inference stages?

**Comment by sh-lee-prml** [comment]
Hi 

I have attached tensorboard loss curves for our TTV v1 model which was trained with LibriTTS-960 dataset. 
we used 4x GPUs with 128 batch size (32 per GPU). 

![285719622-21e3d5cc-199a-4437-b334-adefb0c693f8](https://github.com/sh-lee-prml/HierSpeechpp/assets/56749640/6b21bd95-c20a-4f2f-beae-ebb4c4e9bce3).

**Comment by Pranjalya** [comment]
We have used phonemizer as well, and from past experience, it works decently for Hindi as well. 
Here are my logs:
![image](https://github.com/sh-lee-prml/HierSpeechpp/assets/36627085/8fa4b6b9-c84f-4533-afce-029d33fe6988)

"unintelligible" means like it sounded like it was speaking clearly but nothing related to the text and not in the language. But again, it was just with 20k steps checkpoint.

**Comment by rishikksh20** [comment]
@hayeong0 from how many steps onward we start getting some audible voice when train TTV from scratch ?

**Comment by Pranjalya** [comment]
Just for reference, the audio from 20k steps.

https://github.com/sh-lee-prml/HierSpeechpp/assets/36627085/25cc5bc9-b262-4150-a673-39f6bc6ebca8

**Comment by sh-lee-prml** [comment]
Here is our results from 10k, 20k, 50k, 100k, 200k, 950k. 

I have attached audio for some text and speaker of libritts-test-clean.

[Link](https://drive.google.com/drive/folders/1sWrfTa763CZVb7FjmrFhL8dfi505NXVG?usp=drive_link)

When using LibriTTS dataset, the 10k steps model can synthesize an audible speech.

Thanks!

**Comment by Pranjalya** [comment]
Thank you very much, it helped.
